.github/workflows/ci.yml

```diff
@@ -8,9 +8,9 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - name: "Check out repository"
+      - name: 'Check out repository'
         uses: actions/checkout@v1
-      - name: "Install lunatic"
+      - name: 'Install lunatic'
         run: cargo install --git https://github.com/lunatic-solutions/lunatic
       - name: Install rust
         uses: actions-rs/toolchain@v1
```

Comment:
I don't think we need to change the quoting style here :)

Reply:
VSCode auto format :facepalm: just reverted it now